applications/serializers.py

```diff
@@ -41,14 +41,24 @@ class SubmissionSerializer(serializers.ModelSerializer):
     """ApplicationStepSubmission serializer"""
 
     interview_url = serializers.SerializerMethodField(required=False, allow_null=True)
+    take_interview_url = serializers.SerializerMethodField(
+        required=False, allow_null=True
+    )
 
     def get_interview_url(self, submission):
-        """ Get the interview url for a video submission if of that type"""
+        """Return the results URL for the reviewer or others to view the interview"""
```

Comment:
Can you rename this to `get_result_interview_url` to disambiguate more clearly?

Reply:
I agree this is confusing but I'd like to address this later in the interest of time